markers/sauce.py

```diff
@@ -18,3 +19,11 @@ def pytest_configure(config):
             config.option.markexpr = 'sauce and ({})'.format(config.option.markexpr)
         else:
             config.option.markexpr = 'sauce'
+
+
+def pytest_runtest_setup(item):
+    browser.ensure_browser_open(item)
+
+
+def pytest_runtest_teardown():
```

Comment:
Grr....ok, :) Scratch that, but I would say don't "quit" it, we can always interrogate on setup of the next test and see if we are already having the right browser.